packages/core/src/configuration.ts

```diff
@@ -40,6 +41,8 @@ export interface UserConfiguration {
   applicationId?: string
   internalMonitoringApiKey?: string
   isCollectingError?: boolean
+  enableTracing?: boolean
+  shouldTraceUrl?: (url: string) => boolean
```

Comment:
What pros/cons do you see?

Reply:
To be clear, I'm not saying to replace the function, but to also add a list. In that sense, there is no con (other than having to implement it and adding tests). Pros are that it's much more friendly for a user to provide a list of domains for example, compared to having to implement a logic that they will need to test.

Reply:
For both cases, we will need to add configuration examples in the docs. If we provide a simple example that they can copy/paste, IMO it should be as friendly as the list.
I'd prefer to avoid to maintain multiple options for the same feature.

Reply:
something like `extraDomainsToTrace` could be simpler to understand and should match the need of customers.
any preference?

Reply:
I would prefer the same configuration option as otherwise there would be a conflict between the 2. I'm mostly trying to make onboarding as easy as possible, and a list of domains would definitely be easier to configure. For most cases there won't be a need to configure the rest of the URL, and even less so to programmatically detect what to trace.

Reply:
Do you mean the same configuration as in dd-trace-js? `distributedTracingOriginWhitelist?: (string|RegExp)[]`
If yes, I'd be in favor of `allowedTracingOrigins`. Let's not use whitelist and I don't get the value of distributed.

Reply:
done in 82c8978